pkgs/top-level/all-packages.nix

```
@@ -3173,6 +3173,8 @@ with pkgs;
 
   logseq = callPackage ../applications/misc/logseq { };
 
+  logseq-webapp = callPackage ../applications/misc/logseq/web-app.nix { };
```

It's a different package, so a new package and renaming the Nix file to `default.nix` should be fine. :)

```suggestion
  logseq-webapp = callPackage ../applications/misc/logseq-web/default.nix { };
```

---

Should we not be adding to `pkgs/by-name`?

---

Yep, you're right. Thanks for pointing that out.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md